dc

```diff
@@ -10,6 +10,24 @@ if [ -f .env ]; then
   IFS=$'\n'
   for x in $(grep -v '^#.*' .env); do export $x; done
   IFS=$OLD_IFS
+else
+  # ensure a .env is present
+  touch .env
+fi
+
+if [ -z $(cat .env | grep -i '^SERVER_UID=') ];
+then
+  # add a SERVER_UID to .env if none was found
+  SERVER_UID=$(openssl rand -hex 8)
+  printf "# randomly generated 16 character string that must be unique across servers\nSERVER_UID=$SERVER_UID" >> .env
+  printf "\nIMPORTANT NOTE: added SERVER_UID '$SERVER_UID' to .env file\n\n"
+elif [ -z $(cat .env | grep -E -i '^SERVER_UID=[0-9A-F]{16}') ];
```

Comment:
this takes care of `.env` files where the `SERVER_UID` is malformed, or not set

Reply:
I think it's important that we fill the value if it's `SERVER_UID=` for instance,
still on the fence whether we should assert it's a 16 char lowercase hex string
might make sense to replace it with `grep -E -i '^SERVER_UID=.+'`

Reply:
I personally think this is fine though

Reply:
I don't think we need to assert anything about it. A user should be able to set it to whatever they want. e.g. `eu-ger-1`. It's their responsibility to make it unique if they do it but it should be possible.
We also need this upgrade to not touch any env vars that we set to non-compliant strings.